### DISCUSSION TITLE
Welcome to Cataclysm-DDA Discussions!

### DISCUSSION THREAD

**Discussion by kevingranade** [CleverRaven/Cataclysm-DDA#45912] [General]
## 👋 Welcome!
  We’re enabling discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This new format gives us the opportunity to be more expansive about what kind of content we can host on github.  Brainstorming and incomplete feature suggestions will be welcome in discussions where previously they have been redirected to the forums.  We will be considering migrating some or all of the issues labelled as [Suggestion/Discussion](https://github.com/CleverRaven/Cataclysm-DDA/issues?q=is%3Aissue+is%3Aopen+label%3A%22%3CSuggestion+%2F+Discussion%3E%22) to discussions, but there's no hurry.

Feel free to comment below with an introduction of yourself and your interests 
Feedback about the discussion feature itself would also be welcome.

EDIT: We're adjusting our approach slightly, and restricting new discussion creation (but not posting in discussions) to project members, because the level of traffic this has generated is more than the (mostly non-existent) moderation tools GitHub has made available can easily handle.

**Comment by Mom-Bun** [comment]
Hello! I am Mom Bun!
I've been with Cataclysm for a few years now, and its a game that I have a very soft spot for, as it has kindled a lot of interest for me! I am deeply passionate about the community for this game, and deeply love its design direction. 

You'll probably recognize me from the discord, as I am mostly active there, but I do try to put out features and engage in the github when I feel like I can!

**Comment by ZhilkinSerg** [comment]
I've migrated a couple of issues for test and it worked as a charm.

**Comment by lopho** [comment]
So should this discussion feature be more development focused, or is it sort of a catch all?
Oh, yeah, and "Hello".

**Comment by hjk321** [comment]


**Reply by kevingranade** [reply]
Your options are forums, irc, discord, issue and reddit.

For your use case in particular, this is not content we want on github because it pings multiple developers, is not easily discoverable for people that can provide support, and it sticks around forever.

**Comment by cataclysmsurvivalist** [comment]
How do I start a new discussion for modding? I found an issue and would like the help of more experienced users.

**Reply by ZhilkinSerg** [reply]
Use forums or Discord.

**Comment by fQ7N** [comment]
> ## 👋 Welcome!
> We’re enabling discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> This new format gives us the opportunity to be more expansive about what kind of content we can host on github. Brainstorming and incomplete feature suggestions will be welcome in discussions where previously they have been redirected to the forums. We will be considering migrating some or all of the issues labelled as [Suggestion/Discussion](https://github.com/CleverRaven/Cataclysm-DDA/issues?q=is%3Aissue+is%3Aopen+label%3A%22%3CSuggestion+%2F+Discussion%3E%22) to discussions, but there's no hurry.
> 
> Feel free to comment below with an introduction of yourself and your interests Feedback about the discussion feature itself would also be welcome.
> 
> EDIT: We're adjusting our approach slightly, and restricting new discussion creation (but not posting in discussions) to project members, because the level of traffic this has generated is more than the (mostly non-existent) moderation tools GitHub has made available can easily handle.

Hi! May I know how the game engine displays a vehicle  from the "parts" and "part"?

I am working on assembling a sprite (for truck_swat) and after days of skimming docs, I created a .png where the wheels are on top. This is because a doc stated that the game builds the parts left-to-right, line-by-line. and the wheel appeared last, at the last line of a tile (specified by the json file). True enough, by debugging, I saw the game DOES NOT put wheel on top.

May I know, briefly, how this whole thing actually work programmatically? Thanks!

<img width="360" alt="old" src="https://user-images.githubusercontent.com/109001020/180323531-14ef5502-034d-459f-9337-eaa9c349ff98.png">

**Comment by cresttoothpaste** [comment]
I must  be blind, because i cant find the create new thread button on the suggestion board.

**Reply by anothersimulacrum** [reply]
> EDIT: We're adjusting our approach slightly, and restricting new discussion creation (but not posting in discussions) to project members, because the level of traffic this has generated is more than the (mostly non-existent) moderation tools GitHub has made available can easily handle.

**Reply by xahon** [reply]
@anothersimulacrum how to ask a question on the game then?

**Reply by anothersimulacrum** [reply]
https://github.com/CleverRaven/Cataclysm-DDA/discussions/45912#discussioncomment-1490105

**Comment by kwl01skz** [comment]
The "Survior Mess Kit" has almost lost its meaning. It is described as "It is composed of simple yet durable tools and materials".
However, now the fuel it uses is kerosene. Survivors don't have easy access to kerosene because producing kerosene requires tools for "extraction 3" and we can only make tools for "extraction 2".
Unless we find "Catalytic cracking reactor".

alternatives
1 Reduce the need for extraction tools to manufacture Kerosene.
2 Replace the Survior Mess Kit with a simpler fuel.
3 Add recipe for Catalytic cracking reactor.

**Comment by 321nick** [comment]
Can there be a discussion made for discussing guns in CDDA? i personally think there could be a lot of improvements of the current implementation, especially relating to proficciencies and different models of gun. for example, right now the SPAS-12 and Mossberg 590 are just two different names and descriptions for a gun with identical stats! for a game that values versimillitude highly this doesnt accurately reflect the various differencies in capabilities each gun offers. and this is a game with 3 different fishing devices, so if those have justifiable differences to make them 3 different items im sure the mossberg and the spas 12 are different enough to justify being different items with different stats. but im open to hearing what others have to say.

**Reply by anothersimulacrum** [reply]
An issue would be an appropriate place for that. I can tell you that the Mossberg and SPAS-12 were merged erroneously (sorry), and a PR unmerging them would be accepted.

**Reply by cataclysmsurvivalist** [reply]
Adding to this, I would request that any team responsible for guns remember that most people who play the game are not gun experts, and therefore a few pointers in the item description of firearms to tell amateurs how different they are from each other are often very useful. I understand that in a realistic game like this, studying the real thing in wikis and real-life articles is necessary, but stuff like _"compared to (other rifle), the (this rifle) is easier to aim at the cost of degrading faster"_ is always nice.

**Comment by Lippiece** [comment]
@kevingranade 
Can you address this little [tantrum](https://github.com/CleverRaven/Cataclysm-DDA/pull/64586)? Open-minded community -- cool, but where's that in sticking the head into the sand?

Edit: I've found the answer [here](https://old.reddit.com/r/cataclysmdda/comments/123w2oh/the_end/jdwnnlp/), for someone else stumbling upon this. Locking away the discussion is still, in my opinion, highly stupid per se.

Edit 2: Of course, it's off-topic, I can even make my own discussion.

**Comment by Melganor** [comment]
Hello All, I am somewhat new to CDDA and play mostly magiclysm and is wondering if there is a sound pack with spell FX and the such?

**Comment by Jkesler22** [comment]
Hi!  I have played for years, but just recently started looking into the code and json formatting and such.  I am an IT Sys Admin, and I have a LOT of scripting knowledge, but not much programming experience.  I'd love to help out the project, so I'm going through the guide and figuring things out.  I DO have some ideas that I think need to be discussed, so I'm just getting familiarized with the rest of the community and the site!

**Comment by Alithsko** [comment]
Hey, my name is Alith, but since you can read my user name that's obvious, so I've been an engineering student for a couple years now, basing my career on mechanical ideas, I like to sometimes throw ideas out there to games I'm interested in, been a while since I've played this but recently rediscovered it and remembered why I loved it, only thing holding me back is not knowing much about programing outside of basics, so of course I'm always open to being told that it's not possible, even if it would be a little bit of a let down

**Reply by GuardianDll** [reply]
Json changes require effectively no prior programming knowledge
read <https://github.com/CleverRaven/Cataclysm-DDA/wiki/Guide-to-adding-new-content-to-CDDA-for-first-time-contributors> if you're interested
